### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by jheart-vic**
# we Made a Website with navigation
- we worked with classes
- we stored books in local storage

### CODE REVIEW

File: addbook.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <h3 class="text">Awesome <span class="span">Books</span></h3>
+    <ul>
+      <li><a href="#display-books"> My lists</a></li>
+      <li class="line"><a href="#form"> Add New</a></li>
+      <li><a href="#contact">Contact</a></li>
+    </ul>
```
Comment (KDlamini) [inline addbook.html]: When any of the links are clicked, the main page should change to display `only one` of the three content sections. Currently, when I click on any of the navigation links all sections are still visible on the page when I scroll.

- You can add and remove CSS classes to display or hide the other sections' content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods.
- Each section should seem like it is on its own page when I navigate through the links. When I click `My lists`, only the book list section should be visible. When I click `Add New`, only the add book form section should be visible. When I click `Contact`, only the contact information section should be visible.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi Team! @jheart-vic, @HanningtoneM 🙋‍♀️,  

Great work so far! 💯 
You've done an incredible job creating a complete Awesome Books website with navigation. 👏🏾 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

soclose

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const show = document.querySelector('.show');
+const display = document.querySelector('.display-tex');
+
+navList.addEventListener('click', () => {
+  allBooks.classList.remove('hidden');
+  show.classList.remove('hidden');
+  display.classList.remove('hidden');
+  addBook.classList.add('hidden');
+  contact.classList.add('hidden');
+});
```
Comment (VICTOR-CHUKWU) [inline index.js]: - [ ] currently, when I navigate back to `mylist` link after visiting other links,  the other item still shows in `mylist` page, kindly fix this, you can try this method as I already tried it in the code editor :point_down: 
```
navList.addEventListener('click', () => {
  addBook.classList.add('hidden');
  contact.classList.add('hidden');
  allBooks.classList.remove('hidden');
  show.classList.remove('hidden');
  display.classList.remove('hidden');
});
```add `hidden` class to other DOM content before removing class list from `display`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi Team!, 
## STATUS: CHANGES REQUIRED!

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by m05e5 — APPROVED** [review]
# Hi @jheart-vic , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**